src/components/Configuration.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <div class="configuration">
+    <h2>Select size</h2>
+    <label class="label-rows">Rows: </label>
+    <input type="number" value="" id="rows"><br>
+    <label class="label-columns">Columns: </label>
+    <input type="number" value="" id="columns">
+    <button type="button" @click="getDataBoard()">Create Board</button>
+    <hr><hr><hr><hr><hr>
+    <h2>Join up</h2>
+    <input type="text" id=""><button class="join">go</button>
+  </div>
+</template>
+
+<script>
+import {EventBus} from '@/services/event-bus';
+export default {
+  name: 'configuration',
+  methods: {
+    getDataBoard() {
+      var columns=(document.getElementById("columns").value);
+      var rows=(document.getElementById("rows").value);
```

Comment:
use vuejs features like v-model instead of manual DOM manipulation

Reply:
it was used v-model